game_eggs/steamcmd_servers/arma/ocap2/egg-ocap2.json

```diff
@@ -0,0 +1,77 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2022-03-31T20:58:04+00:00",
+    "name": "OCAP2",
+    "author": "webmaster@arcomm.co.uk",
+    "description": "OCAP is an Arma 3 mission recording suite. It consists of a serverside addon and a packaged web server executable that allows for after-action review and better insight into what really happened.",
+    "features": null,
+    "images": [
+        "ghcr.io\/arcomm\/ocap2-pterodactyl:master"
```

Comment:
Should the existing list of images not contain what's required to run the software, please open a pull request to expand them as informed in [CONTRIBUTING.md](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md#general-rules). We can't accept images controlled by someone else.

The install script should handle downloading the files, instead of a Docker image.